### PR TITLE
i#6136: Add AVX-512 VPOPCNTDQ support.

### LINKED ISSUE #6136: OpenMPI crashes under drrun due to invalid opcode
**Describe the bug**
OpenMPI crashes under drrun because DR disassembler does not understand VPOPCNTQ.

**To Reproduce**
Just running mpirun under drrun results in a crash due to invalid opcode.
```
$ mpirun --version
mpirun (Open MPI) 4.1.4

$ drrun -- mpirun
...
<Invalid opcode encountered>
...
Illegal instruction (core dumped)
```

Running with logs enabled shows this as the bad instruction.
```
SYSLOG_WARNING: Invalid opcode encountered
Error decoding 0x00007ffdf7495150 == 0x62 0xf2 0xfd 0x28 0x55 0xc 0xfa
```
objdump shows that this is vpopcntq which requires AVX512_VPOPCNTDQ which was introduced in Ice Lake

```
62 f2 fd 28 55 0c fa    vpopcntq (%rdx,%rdi,8),%ymm1
```

**Expected behavior**
No crash

**Screenshots or Pasted Text**

**Versions**
drrun version 9.93.19518 -- build 0 (latest version available)

**Additional context**

### PR DESCRIPTION
**Pull request by khuey**
Adds the new AVX512_VPOPCNTDQ instructions, which were present on certain Xeon Phis but broadly introduced in Ice Lake.

Fixes #6136

### CODE REVIEW

**Comment by khuey** [comment]
NB: There's also VPOPCNTB/W but they're part of AVX512_BITALG so I haven't included them here.

File: third_party/binutils/test_decenc/drdecode_decenc_x86_64.expect [hunk 1]
```diff
@@ -102541,7 +102617,7 @@ test_x86_64_s:
  7b 00 00 00
  62 e2 7d 21 91 84 c5 vpgatherqd 0x0000007b(%rbp,%ymm16,8), %ymm16 {%k1} {%k1}
  7b 00 00 00
- 62 f1 7c 48 28 05 00 vmovaps <rel> 0x000000001006ffa6, %zmm0 {%k0}
+ 62 f1 7c 48 28 05 00 vmovaps <rel> 0x000000001007015a, %zmm0 {%k0}
```
Comment (derekbruening) [inline third_party/binutils/test_decenc/drdecode_decenc_x86_64.expect:102620]: What caused this change?

**Review by derekbruening — APPROVED** [review]
Thank you for the PR.

File: third_party/binutils/test_decenc/drdecode_decenc_x86_64.expect (hunk 1 above)
Comment (khuey) [reply third_party/binutils/test_decenc/drdecode_decenc_x86_64.expect:102620]: It's a pc-relative operand and the address of the instruction changed when I removed the #ifdef DISABLED_UNTIL_BUG_1312_IS_FIXED around avx512_vpopcntdq.s

**Comment by khuey** [comment]
I don't understand why the merge queue keeps bouncing this.

**Pull request merged**